hw/top_earlgrey/doc/design/README.md

```diff
@@ -540,6 +540,12 @@ The first EDN instance, `u_edn0` is intended to be configured to deliver entropy
 
 <!-- END CMDGEN -->
 
+#### Open drain / virtual open drain
+
+Pads of type *BidirOd* have true hardware support for open drain termination: when *open drain* is enabled in pinmux's attributes for such a pad, the PMOS transistor that would otherwise pull the output high for a logic 1 is disabled and there is no parallel connection to the supply rail; thus for a logic 1 the pad then has an impedance equal to the output impedance of the CMOS transistor pair (while for a logic 0 the pad is still pulled low by the NMOS transistor).
```

Comment:
Could you clarify whether the voltage of a pad in this true open drain mode is allowed to go above the supply rail (to 5 volts, maybe)?  I realize that this will depend on vendor logic and belong to NuvoTitan documentation, but a hint here would be nice.